backend/models/comment.js

```diff
@@ -20,4 +20,11 @@ commentSchema.method("update", function(updates, callback) {
   Object.assign(this, updates, {updatedAt: new Date()});
   this.save(callback);
 });
-module.exports = mongoose.model('Comment', commentSchema)
+commentSchema.statics.getComment = function(request, response, next, id) {
```

Comment:
This is some interesting code, but I haven't delved deeper into mongo. I would appreciate it if you can explain to me what `[Schema].statics.[method]` means :)